code/tests/atmos/test_ventcrawl.dm

```diff
@@ -7,14 +7,16 @@
 /datum/milla_safe/ventcrawl_test_setup
 
 /datum/milla_safe/ventcrawl_test_setup/on_run(datum/game_test/ventcrawl/test)
+	// I'm sure get_area_turfs is totally deterministic and this will never go wrong
+	var/turf/run_loc_bottom_left = test.available_turfs[1]
```

Comment:
Why not add some landmarks? Might help make things more deterministic

Reply:
Done, PTAL